### DISCUSSION TITLE
Upgrade Laravel 10

### DISCUSSION THREAD

**Discussion by bhargavraviya** [nafiesl/silsilah#113] [General]
- Laravel Upgrade 10 Version
- Bootstrap 5 Design
- New Auth of Laravel UI
- Backup Management (Coming Soon)

Please check this link [https://github.com/raviyatechnical/genealogy](https://github.com/raviyatechnical/genealogy) 
if needed then merge your reps

@nafiesl

**Comment by nafiesl** [comment]
Hi @bhargavraviya, I have quick-checked your repo. Looks like you were doing like so:

1. Setup a fresh Laravel 10
2. Copy and paste most of the code from this repository.

And checking the web content:

1. The view does not look as neat as I expected (you can see the difference below).

    <details>
    <summary>CSS does not look neat</summary>
    
    ### From your repo
    
    ![screen_2023-12-10_002](https://github.com/nafiesl/silsilah/assets/8721551/c540ca11-e25b-4f23-a693-fa270b85798e)
    
    ### From the original repo
    
    ![screen_2023-12-10_003](https://github.com/nafiesl/silsilah/assets/8721551/45c527b9-f2c9-4775-8fe8-2e7566c8e38c)
    </details>
    I am not sure if this repo is still in progress or you have already finished working on it.

2. You were missing the lang switcher feature, where the original repo has 3 languages.

3. Since you were copy-paste most of the code from this repo. We need to discuss your readme.MD content:
    - **The LICENSE**
       You already know that the original source code of the Silsilah project uses MIT license. I see you were using MIT license on the repo, but not using the original LICENSE text.

      For your information, MIT license **allows you** to modify and use the project for commercial, but you **must include** the [original LICENSE text](https://github.com/nafiesl/silsilah/blob/master/LICENSE) file (which contains the original author name) on your modified source code. Moreover, you published the code publicly on your github profile.

       So I encourage you to **add the original LICENSE text on** your repository.

       If you have doubts about the MIT license rule, you can refer to this site https://www.tldrlegal.com/license/mit-license (see the MUST section)

    - **Inspere by Project https://github.com/nafiesl/silsilah**
      I believe it is not correct that you were "inspired" by the project, but you just modified the original project and then uploaded it on your github profile. So I suggest you to change that section like so:

      > ## The Original Project
      > This is the modified version of the project: [https://github.com/nafiesl/silsilah](https://github.com/nafiesl/silsilah). Here are my improvements:
      > - (mention your changes here) 

By doing so, people who read your repo will know that:
1. You can improve someone else work professionally.
2. You respect the original author or the project.
3. Your improvements from the original project (what's missing then you improve it).

Thanks for your time and attention for the project @bhargavraviya. I appreciated it.

**Comment by bhargavraviya** [comment]
- it private reps only you can see that code that purpose public. now it is private reps
- I'm Working on upgrade 10
- I'm working on upgrading some features pending according to the new Laravel 10

@nafiesl 

[https://github.com/bhargavraviya/silsilah/tree/upgrade/laravel-v10](https://github.com/bhargavraviya/silsilah/tree/upgrade/laravel-v10)
